infra/lib/api-service.ts

```diff
@@ -133,6 +133,8 @@ export function createAPIService(
   // RW grant for Dynamo DB
   dynamoDBTokenTable.grantReadWriteData(fargateService.taskDefinition.taskRole);
 
+  cdaToVisualizationLambda.grantInvoke(fargateService.taskDefinition.taskRole);
```

Comment:
This was previously done on the code that creates the lambda. This created a circular dependency that required the additional config variable. Moving here is a single direction dependency.